go/libkb/resolve.go

```diff
@@ -171,6 +171,7 @@ func (r *Resolver) resolveURLViaServerLookup(au AssertionURL, input string, with
 
 	var them *jsonw.Wrapper
 	if them, res.err = ares.Body.AtKey("them").ToArray(); res.err != nil {
+		res.err = NotFoundError{fmt.Sprintf("User not found for query: %q", input)}
```

Comment:
See [this API call](https://keybase.io/_/api/1.0/user/lookup.json?username=thurnish41ey)